### ISSUE TITLE
Can u give some train data !!!!

### ISSUE THREAD

**Issue by anthonyyuan** [leehomyc/Photo-Realistic-Super-Resoluton#3]
train_folder=/Users/anthonyyuan/Desktop/src/ccc/Photo-Realistic-Super-Resoluton-master/train_folder model_folder=/Users/anthonyyuan/Desktop/src/ccc/Photo-Realistic-Super-Resoluton-master/your_save_folder th run_sr.lua
{
  ntrain : inf
  beta1 : 0.9
  name : "super_resolution"
  dataset : "folder"
  niter : 250
  lr : 0.001
  model_folder : "/Users/anthonyyuan/Desktop/src/ccc/Photo-Realistic-Super-Resoluton-master/your_save_folder"
  batchSize : 32
  scale : 4
  nThreads : 4
  gpu : 1
  train_folder : "/Users/anthonyyuan/Desktop/src/ccc/Photo-Realistic-Super-Resoluton-master/train_folder"
  loadSize : 96
}
Starting donkey with id: 1 seed: 1
table: 0x0fe69158
Starting donkey with id: 4 seed: 4
table: 0x0fe89518
Starting donkey with id: 2 seed: 2
table: 0x0fea9328
Starting donkey with id: 3 seed: 3
table: 0x0fec9158
Creating train metadata
table: 0x0fea0670
class pics
running "find" on each class directory, and concatenate all those filenames into a single file containing all image paths for a given class
Creating train metadata
table: 0x100c1508
class pics
running "find" on each class directory, and concatenate all those filenames into a single file containing all image paths for a given class
Creating train metadata
table: 0x0fd6a388
class pics
running "find" on each class directory, and concatenate all those filenames into a single file containing all image paths for a given class
Creating train metadata
table: 0x1010bca8
class pics
running "find" on each class directory, and concatenate all those filenames into a single file containing all image paths for a given class
/tmp/lua_ROTz68: line 1: gfind: command not found
/tmp/lua_Ya6sYp: line 1: gfind: command not found
/tmp/lua_8GnZwL: line 1: gfind: command not found
/tmp/lua_pJm6Y2: line 1: gfind: command not found
now combine all the files to a single large file
now combine all the files to a single large file
now combine all the files to a single large file
now combine all the files to a single large file
load the large concatenated list of sample paths to self.imagePath
load the large concatenated list of sample paths to self.imagePath
load the large concatenated list of sample paths to self.imagePath
load the large concatenated list of sample paths to self.imagePath
sh: gwc: command not found
sh: gwc: command not found
sh: gwc: command not found
sh: gwc: command not found
/Users/anthonyyuan/torch/install/bin/luajit: ...honyyuan/torch/install/share/lua/5.1/threads/threads.lua:183: [thread 4 callback] .../Photo-Realistic-Super-Resoluton-master/data/dataset.lua:206: attempt to perform arithmetic on a nil value
stack traceback:
    .../Photo-Realistic-Super-Resoluton-master/data/dataset.lua:206: in function '__init'
    ...s/anthonyyuan/torch/install/share/lua/5.1/torch/init.lua:91: in function <...s/anthonyyuan/torch/install/share/lua/5.1/torch/init.lua:87>
    [C]: in function 'dataLoader'
    ...tic-Super-Resoluton-master/data/donkey_folder_supres.lua:76: in main chunk
    [C]: in function 'dofile'
    ...ccc/Photo-Realistic-Super-Resoluton-master/data/data.lua:37: in function <...ccc/Photo-Realistic-Super-Resoluton-master/data/data.lua:27>
    [C]: in function 'xpcall'
    ...honyyuan/torch/install/share/lua/5.1/threads/threads.lua:234: in function 'callback'
    ...nthonyyuan/torch/install/share/lua/5.1/threads/queue.lua:65: in function <...nthonyyuan/torch/install/share/lua/5.1/threads/queue.lua:41>
    [C]: in function 'pcall'
    ...nthonyyuan/torch/install/share/lua/5.1/threads/queue.lua:40: in function 'dojob'
    [string "  local Queue = require 'threads.queue'..."]:13: in main chunk
stack traceback:
    [C]: in function 'error'
    ...honyyuan/torch/install/share/lua/5.1/threads/threads.lua:183: in function 'dojob'
    ...honyyuan/torch/install/share/lua/5.1/threads/threads.lua:264: in function 'synchronize'
    ...honyyuan/torch/install/share/lua/5.1/threads/threads.lua:142: in function 'specific'
    ...honyyuan/torch/install/share/lua/5.1/threads/threads.lua:125: in function 'Threads'
    ...ccc/Photo-Realistic-Super-Resoluton-master/data/data.lua:25: in function 'new'
    run_sr.lua:32: in main chunk
    [C]: in function 'dofile'
    ...yuan/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:145: in main chunk
    [C]: at 0x010f210bc0

**Comment by leehomyc** [comment]
I would suggest trying the ImageNet dataset, and just set t_folder="ILSVRC2015/Data/CLS-LOC/train" where it contains the ImageNet sub-folders.

**Comment by HuO50** [comment]
does it need all imagenet image to train this model? thanks for coding
